test/extended/prometheus/prometheus_builds.go

```diff
@@ -209,10 +215,10 @@ func labelsWeWant(sample *model.Sample, labels map[string]string) bool {
 }
 
 func valueWeWant(sample *model.Sample, tc metricTest) bool {
-	//NOTE - we could use SampleValue has an Equals func, but since SampleValue has no GreaterThan,
+	//NOTE - we could use SampleValue has an Equals func, but since SampleValue has no GreaterThanEqual,
 	// we have to go down the float64 compare anyway
-	if tc.greaterThan {
-		return float64(sample.Value) > tc.value
+	if tc.greaterThanEqual {
+		return float64(sample.Value) >= tc.value
```

Comment:
wasn't the issue that we're getting a non-zero value when we expect zero?  how does this help that?

Reply:
greaterThan was set to false in those cases, so we fell back to the == check below

if you noticed I set greaterThanEqual to true for cancelled / failed

Reply:
got it.